src/style.css

```
@@ -0,0 +1,30 @@
+.body {
+  margin: 0;
+}
+
+.stars :nth-child(-n + 4) {
+  margin-right: 4px;
+
+}
+
+.stars_star {
+  background-image: url("images/star.svg");
+  height: 16px;
+  width: 16px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star--active {
+  background-image: url("images/star-active.svg");
+}
```

The block with stars--N modifier should have exactly N first stars active, according to [requirements](https://github.com/mate-academy/layout_stars#the-task)

Please try to make it work with `nth-child` pseudoclass

```suggestion
.stars--1 .stars__star:nth-child(-n+1), {
  ... more selectors to target every line of stars ... 
  background-image: url("./images/star-active.svg");
}
```